build.gradle

```diff
@@ -46,7 +46,7 @@ repositories {
 }
 
 ext {
-    opendistroVersion = '1.1.0'
+    opendistroVersion = '1.2.0'
```

Comment:
The same for the version in `gradle.properties`? https://github.com/opendistro-for-elasticsearch/sql/pull/91/files#diff-503f218d646c10f484fdc9d6315bf2e3